bayeux.go

```diff
@@ -95,13 +105,31 @@ type AuthenticationParameters struct {
 
 // Bayeux struct allow for centralized storage of creds, ids, and cookies
 type Bayeux struct {
-	creds Credentials
-	id    clientIDAndCookies
+	creds  Credentials
+	id     clientIDAndCookies
+	status Status
```

Comment:
Does the `Status` need to be exposed?

Reply:
We don't really need it exposed.
Status was already exposed, I am just reusing it to get connection information from bayeux object.

Reply:
Ok, let's hide it then if it isn't necessary.

Reply:
Okay